### PR TITLE
Add development and build info

### LINKED ISSUE #15: Mention build process in Readme
@yedhink _a

### PR DESCRIPTION
**Pull request by yedhink**
Fixes #15 

@vinay0x _a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,21 @@ Import the styles in your `application.scss` file by doing
...
+```
+You can create new components in the `lib/components` and add a reference to the same in `src/index.js`.  
+
+Run the following to test it out:-
+```
+yarn start
+```
+
+## Building
+Once everything is done. Run `yarn build` to build new bundle files and **increment the version number** in `package.json`
```
Comment (vinay0x) [inline README.md:32]:
```suggestion
Run `yarn build` to build new bundle files and **increment the version number** in `package.json`. Make sure to commit and push the changes to Github. In the host application, run `yarn add https://github.com/bigbinary/nitroui.git` to update it to the latest version.
```

File: README.md [hunk 2]
```diff
@@ -16,6 +16,21 @@ Import the styles in your `application.scss` file by doing
...
+Install all the dependencies first:-
+```
+yarn
+```
+You can create new components in the `lib/components` and add a reference to the same in `src/index.js`.  
+
+Run the following to test it out:-
+```
+yarn start
+```
```
Comment (vinay0x) [inline README.md:29]:
```suggestion
Running the `yarn start` command starts a CRA app which resides in `src` folder. Use this application to test out changes. Note that nothing in the `src` folder will be bundled with NitroUI.
```

File: README.md [hunk 3]
```diff
@@ -16,6 +16,21 @@ Import the styles in your `application.scss` file by doing
 import 'remixicon/fonts/remixicon.css'
 ```
 
+## Development
+Install all the dependencies first:-
```
Comment (vinay0x) [inline README.md:20]:
```suggestion
Install all the dependencies by running:
```

File: README.md [hunk 4]
```diff
@@ -16,6 +16,21 @@ Import the styles in your `application.scss` file by doing
...
+```
+yarn
+```
+You can create new components in the `lib/components` and add a reference to the same in `src/index.js`.
```
Comment (vinay0x) [inline README.md:24]:
```suggestion
You can create new components in the `lib/components` and export them from`lib/index.js`.  
```

**Review by vinay0x — CHANGES_REQUESTED** [review]
@yedhink _a

**Comment by yedhink** [comment]
@vinay0x _a

Have made the requested changes.

**Pull request merged**